src/foundry/client-esm/nue/tour.d.mts

```diff
@@ -1,9 +1,276 @@
+import type { ValueOf } from "fvtt-types/utils";
+
 /**
  * A Tour that shows a series of guided steps.
- * @remarks TODO: Stub
  */
-declare class Tour {}
+declare class Tour {
+  /**
+   * Construct a Tour by providing a configuration.
+   * @param config  - The configuration of the Tour
+   * @param options - Additional options for configuring the tour
+   *                  (default: `{}`)
+   */
+  constructor(config: Tour.Config, options?: Tour.ConstructorOptions);
+
+  static STATUS: Readonly<{
+    UNSTARTED: "unstarted";
+    IN_PROGRESS: "in-progress";
+    COMPLETED: "completed";
+  }>;
+
+  /**
+   * Indicates if a Tour is currently in progress.
+   */
+  static get tourInProgress(): boolean;
+
+  /**
+   * Returns the active Tour, if any
+   */
+  static get activeTour(): Tour | null;
+
+  /**
+   * Handle a movement action to either progress or regress the Tour
+   * @param movementDirections - The Directions being moved in
+   */
+  static onMovementAction(movementDirections: string[]): boolean | void;
+
+  /**
+   * Configuration of the tour. This object is cloned to avoid mutating the original configuration.
+   */
+  config: Tour.Config;
+
+  /**
+   * The HTMLElement which is the focus of the current tour step.
+   */
+  targetElement: HTMLElement | undefined | null;
+
+  /**
+   * The HTMLElement that fades out the rest of the screen
+   */
+  fadeElement: HTMLElement | undefined;
+
+  /**
+   * The HTMLElement that blocks input while a Tour is active
+   */
+  overlayElement: HTMLElement | undefined;
```

Comment:
The types here seems correct. Nice! I'd put in #3258 that they don't mention `undefined` and `null` here.